src/types.rs

```diff
@@ -214,8 +186,37 @@ pub enum Event {
     Teleported,
 }
 
-pub type TechId = String;
-pub type VictimClass = String;
-pub type KillerWeapon = String;
-pub type ResearchId = String;
-pub type ModName = String;
+#[derive(Serialize_repr, Deserialize_repr, PartialEq, Eq, Debug, Copy, Clone)]
+#[repr(u8)]
+pub enum Team {
+    None = 0,
+    Marines = 1,
+    Aliens = 2,
+}
```

Comment:
Most of the time, 0 is not a valid option for the team. In fact, the only time 0 is valid is for `winningTeam` afaik, so I think it would be better to make a separate `enum` for that.